.github/workflows/ci.yml

```diff
@@ -42,6 +42,10 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.pypi_password }}
+      - uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
We didn't set a token in our secrets and this is not necessary for public repos

see for example
https://github.com/JDASoftwareGroup/kartothek/blob/4e4b5e0f827d8b303085ef0dba6ca57a6cb3b3dc/.github/workflows/ci.yml#L137-L142